### ISSUE TITLE
Recommendation of use of PHP 7.2

### ISSUE THREAD

**Issue by ajgw** [woocommerce/woocommerce#20465]
With WC 3.4.x there is under WC/status a recommendation for min. of PHP 7.2 for it. But as I saw not all WC extensions from WooCommerce.com marketplace are tested to latest version of PHP. This could lead some critical bugs and blank screens. Be careful in case of recommending such.

**Comment by mikejolley** [comment]
Hi,

7.2 is also recommended by WordPress https://en-gb.wordpress.org/about/requirements/

We don't feel we should not recommend 7.2 for this reason - I would say, if you do come across a plugin which is not tested/compatible with 7.2 to report it to the author/helpdesk, but I think it's unlikely 7.2 would cause anything worse than a warning/notice.

Thanks

**Comment by Shantel444** [comment]
Interesting information! I've recently found out that PHP is pretty popular nowadays and offers many great solutions. Also I've already explored the subject of the wages and found a good article on [https://diceus.com/programmers-salary-country-expertise-php/](https://diceus.com/programmers-salary-country-expertise-php/). I should say, all this information persuaded me that PHP is worth learning. And I'm sure your posts will help me. Thanks!
